manifests/input.yaml

```diff
@@ -66,31 +66,41 @@ work/input/RYF.vas.1990_1991.nc:
   hashes:
     binhash: 17224eecb745b6ce72604b41201c1fe7
     md5: 88dc8c70338bbc8f5efc48ed0009a99f
-work/input/access-om2-1deg-ESMFmesh.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/access-om2-1deg-ESMFmesh.nc
-  hashes:
-    binhash: b76768044950ac2a165e3c2cd70e2f56
-    md5: 670568bbb1f7f14c534b37ec895af118
-work/input/access-om2-1deg-nomask-ESMFmesh.nc:
-  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/1deg/share/access-om2-1deg-nomask-ESMFmesh.nc
-  hashes:
-    binhash: 25269214daf2808dbf536f434b707b71
-    md5: d40d8512d683a089b6feb343d903ad71
+work/input/access-om2-025deg-ESMFmesh.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/share/access-om2-025deg-ESMFmesh.nc
+  hashes:
+    binhash: 62206020c8390971f3f70ff6d1e2eab5
+    md5: 9f5055fb3d36eecd964f5f1c98dd6a80
+work/input/access-om2-nomask-025deg-ESMFmesh.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/share/access-om2-nomask-025deg-ESMFmesh.nc
+  hashes:
+    binhash: dbaa2ffa55d29117d433ea1e7cd3afbd
+    md5: 3ca48372532f9b9b7890936d600ac509
+work/input/chl.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/mom/chl.nc
+  hashes:
+    binhash: 3079228e9abaab3193b52b3109bd068b
+    md5: 99b68aec777292d746e4d93c2b976189
+work/input/global_gx1v5_grid.nc:
+  fullpath: /g/data/ik11/inputs/access-om3/0.x.0/025deg/cice/global_gx1v5_grid.nc
```

Comment:
Please delete this from the input folder, its not used.

Reply:
Update input and restart yaml files in this commit 058e5f7